.github/workflows/goreleaser.yml

```diff
@@ -43,3 +45,21 @@ jobs:
         KO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         USER: ${{ github.actor }}
         EMAIL: ${{ github.actor}}@users.noreply.github.com
+
+    - name: Set up Helm
+      uses: azure/setup-helm@v4
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Login to registry
+      uses: docker/login-action@v3
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Package and push chart
+      run: |
```

Comment:
in ks repo I've seen we have 
```
chartVersion=$(echo ${{ github.ref_name }} | cut -c 2-)
```
don't we need it here as well?

Reply:
ok, changed, I guess it is to remove the `v`.
I used kubeflex code which does not remove it.